homeassistant/components/neato.py

```diff
@@ -32,6 +32,7 @@
 }, extra=vol.ALLOW_EXTRA)
 
 STATES = {
```

Comment:
Where is this used? I can't find usage in any of the platforms.

Reply:
I just took a closer look, I don't think these `STATES` were ever used (not even imported).  They were actually added in `switch.neato` when the component was first created.  I will remove this and test it out before submitting a change.

Reply:
Looks like things are ok, shall I submit it to this PR OR open another one to remove the unused `STATES`?

Reply:
I think we can do it here.